adafruit_circuitplayground/circuit_playground_base.py

```diff
@@ -48,6 +48,7 @@
 import board
 import busio
 import digitalio
+import audiomp3
```

Comment:
I will test it out today on a CPX and make the import conditional if needed. Although I wonder if there will still be a desire to have this feature if it can only work on CPB and not CPX, maybe it's best to leave it out of this library if that is the case. 

I'll poke around some today and see what I come up with.